srcpkgs/mercury/template

```diff
@@ -0,0 +1,150 @@
+# Template file for 'mercury'
+pkgname=mercury
+version=22.01
+revision=1
+wrksrc="mercury-srcdist-${version}"
+hostmakedepends="flex bison gettext texinfo"
+depends="mercury-libs>=${version}_${revision}"
+short_desc="Logic/Functional Programming Language"
+maintainer="Emily McDonough <emily@alaskanemily.net>"
+license="GPL-2.0-only"
+homepage="https://mercurylang.org/"
+distfiles="http://dl.mercurylang.org/release/mercury-srcdist-${version}.tar.xz"
+checksum=4ab0734aaa66808c91ffe7f54c7d818bdc4f0247aa255932e1e3d50610db4150
+nocross="Mercury cannot be cross-compiled to different architectures"
```

Comment:
We usually set $nocross to yes or set it to a reason why it can't be cross compiled. This sounds to me like this can't be cross compiled because it can't be cross compiled.